grafana/grafana
# loki: backend-mode: add stats support

when using the loki-data-source, the Loki database returns statistics, which we display in the query-inspector panel.
this pull-request handles this in the backend-mode

NOTES:
- i added some code to try-to-detect that there are no statistics returned, to avoid setting lots of zero-value stats-fields
- i adjusted the stats-values in the streams-sample-json to make every stats-value unique. this should help us detect bugs in the future

how to test:
- enable the `lokiBackenMode` feature flag
- run a loki query in explore
- open the query inspector, and verify that you see additional stats
- now run a logs-producing loki query
- verify that the logs panel shows the `Total bytes processed:` info
